OrbitGl/TimeGraph.h

```diff
@@ -107,7 +108,8 @@ class TimeGraph {
   void SetCanvas(GlCanvas* canvas);
   [[nodiscard]] GlCanvas* GetCanvas() { return canvas_; }
   void SetFontSize(int font_size);
-  [[nodiscard]] int GetFontSize() { return GetTextRenderer()->GetFontSize(); }
+  [[nodiscard]] int GetFontSize() { return lround((GParams.font_size) * layout_.GetScale()); }
+  [[nodiscard]] int GetOverlayFontSize() { return lround(GParams.font_size); }
```

Comment:
I think it might be worth it to change the type of font_size in Params.h to uint32_t instead of float also.